test/ruby/test_gc_compact.rb

```diff
@@ -373,7 +373,7 @@ def add_ivars
 
       Fiber.new {
         $ary = OBJ_COUNT.times.map { Foo.new }
-        $ary.each(&:add_ivars)
+        $ary.reverse_each(&:add_ivars)
```

Comment:
Seems unnecessary to change this?

Reply:
You're right. I remember it was failing at one point, but I can't reproduce that anymore. So I removed it https://github.com/ruby/ruby/pull/9533/commits/2e2cd3f60c5ff8a4b72414e96275b687045342f3.